<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QDialog" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="send_chat">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>400</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>传送（S）</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="QQface">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/lenovo/Desktop/好未来/IM_QQ-master/IM_QQ-master/client/face/2.gif</normaloff>C:/Users/lenovo/Desktop/好未来/IM_QQ-master/IM_QQ-master/client/face/2.gif</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="filetrans">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <activeon>C:/Users/lenovo/Desktop/好未来/IM_QQ-master/IM_QQ-master/client/face/fileopen.png</activeon>
       </iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
